CL/cl_gl.h

```diff
@@ -152,6 +152,16 @@ typedef CL_API_ENTRY cl_int (CL_API_CALL *clGetGLContextInfoKHR_fn)(
     void *                        param_value,
     size_t *                      param_value_size_ret);
 
+/* 
+ *  cl_khr_gl_event extension
+ */
+#define CL_COMMAND_GL_FENCE_SYNC_OBJECT_KHR     0x200D
+
+extern CL_API_ENTRY cl_event CL_API_CALL
+clCreateEventFromGLsyncKHR(cl_context context,
+                           cl_GLsync  sync,
+                           cl_int *   errcode_ret) CL_EXT_SUFFIX__VERSION_1_1;
```

Comment:
`s/CL_EXT/CL_API/`?

I know this is happening in a different PR, but this will likely get missed if the other PR lands first.

Reply:
Changed my mind - will make @kpet rebase the other PR and handle it there instead :-)

Reply:
Makes sense given the change needed on the other PR.